cs-verifiable-encryption/README.md

```diff
@@ -0,0 +1,135 @@
+- Feature Name: cs-verifiable-encryption
+- Start Date: 2019-07-24
+- RFC PR: 
+- Ursa Issue: 
+- Version: 1
+
+# Summary
+[summary]: #summary
+
+[Camenisch-Shoup verifiable encryption](https://www.shoup.net/papers/verenc.pdf) is a protocol for proving "properties" 
+about encrypted data. The "properties" are discrete log relations and the encryption is public key encryption. The party 
```

Comment:
"are discrete log relations": relations between *what*, exactly? I'm assuming relations between the values of plaintext fields?

Reply:
discrete log only makes sense for numbers. Are you hinting that i should mention something about credentials?